packages/elements-core/src/components/Docs/HttpOperation/HttpOperation.tsx

```diff
@@ -16,88 +16,93 @@ import { Responses } from './Responses';
 
 export type HttpOperationProps = DocsComponentProps<IHttpOperation>;
 
-const HttpOperationComponent = React.memo<HttpOperationProps>(({ className, data, headless, uri }) => {
-  const info = React.useContext(ActiveInfoContext);
-  const mocking = React.useContext(MockingContext);
-  const isDeprecated = !!data.deprecated;
-
-  const [responseMediaType, setResponseMediaType] = React.useState('');
-  const [responseStatusCode, setResponseStatusCode] = React.useState('');
-  const [requestBodyIndex, setTextRequestBodyIndex] = React.useState(0);
-
-  const httpServiceUri = uri && getServiceUriFromOperation(uri);
-
-  const securitySchemes = flatten(data.security);
+const HttpOperationComponent = React.memo<HttpOperationProps>(
```

Comment:
This file is mostly the same. The only change is adding two props and using them in TryIt section